packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -79,7 +84,7 @@ export class VerticalStackedBarChartBase extends React.Component<
   IVerticalStackedBarChartState
 > {
   private _points: IVerticalStackedChartProps[];
-  private _dataset: IDataPoint[];
+  private _dataset: IVerticalStackedBarDataPoint[];
```

Comment:
>IVerticalStackedBarDataPoint [](http://example.com/codeflow?start=20&length=28)

Will this break existing users?

Reply:
It is type of IDataPoint with x redeclared as Date type also. It will not break existing users.This is done because IDataPoint is used in other charts.